.github/CODEOWNERS

```diff
@@ -673,6 +697,11 @@
 # PRLabel: %Storage
 /sdk/storage/Microsoft.Azure.WebJobs.*/         @seanmcc-msft @amnguye @jaschrep-msft @tg-msft @fabiocav @mathewc
 
+/sdk/storagecache/*.yml                         @seanmcc-msft @amnguye @jaschrep-msft @tg-msft
```

Comment:
Should we do `/sdk/storage*` above?

Reply:
@ArthurMa1978:

Should I assign the same set of owners to all these directories, in `/sdk` ?
![image](https://user-images.githubusercontent.com/4429827/216450288-6e320f85-f357-430a-839c-ab909df176a6.png)

Per @jsquire:

> To my knowledge, the Storage team owns all storage client libraries in our repository. If those paths are management libraries, which I’m not sure is the case, those should go to @ArthurMa1978 (...) who own the management plane all-up.